packages/styles/src/components/form-check.scss

```diff
@@ -93,10 +94,25 @@
       border-color: rgba(var(--post-contrast-color-rgb), 0.4) !important;
       color: rgba(var(--post-contrast-color-rgb), 0.4) !important;
     }
+
+    .form-check-sm & {
+      height: form-check.$form-check-input-size-sm;
+      width: form-check.$form-check-input-size-sm;
+
+      &[type='radio'] {
+        &:checked::after {
+          border-width: spacing.$size-line;
+        }
+      }
+    }
   }
 
   &-label {
     flex: 1;
+
+    .form-check-sm & {
+      font-size: type.$font-size-12;
```

Comment:
Eventually, I think it would be nice to just define `<body class="small/intranet">` and every component inherits font sizes and acts accordingly. It's not quite developer friendly to have to specify on every single element that you want small variants.

@imagoiq, should we account for this now or update font-sizes and small/large differentiation later?

Reply:
I think that it's a good idea. We may want both solution to be sure to be flexible enough. As this touches as well other components, we should do this in another ticket.